messaging/src/main/java/org/axonframework/eventhandling/MergedTrackingToken.java

```diff
@@ -165,6 +165,14 @@ public boolean covers(TrackingToken other) {
         return lowerSegmentToken.covers(other) && upperSegmentToken.covers(other);
     }
 
+    @Override
```

Comment:
Essentially the same question as on the `GlobalSequenceTrackingToken` - was the override mandatory? And, yet again, perhaps it is, as for AF5 I think it's best if this new method we're adding does **not** have a `default` implementation at all.

Reply:
Here it's needed, because if one of the lower/upper segment is a GapAwareTrackingToken then covers != equalsLatests.